content/Watsonx/WatsonxGov/Level-4/202.md

```diff
@@ -0,0 +1,180 @@
+---
+title: '202: Deploy up a watsonx.ai model [optional]'
+timeToComplete: 45
+updated: 2024-06-26
+---
+
+# 202: Deploy up a watsonx.ai model [optional]
+
+In this lab, we will deploy a watsonx model.
+This lab can be optional as it is only needed to complete the **Evaluate the watsonx model** step of the **Lab 207: Govern generative models** lab.
```

Comment:
Add link to lab 207: `**[Lab 207: Govern generative models](/watsonx/watsonxgov/level-4/207)**`